src/main/java/gov/usgs/traveltimeservice/TravelTimeController.java

```diff
@@ -0,0 +1,45 @@
+package gov.usgs.traveltimeservice;
+
+import gov.usgs.processingformats.TravelTimeException;
+import gov.usgs.processingformats.TravelTimePlotRequest;
+import gov.usgs.processingformats.TravelTimeRequest;
+import gov.usgs.traveltime.TTService;
+import io.micronaut.context.annotation.Value;
+import io.micronaut.http.HttpResponse;
+import io.micronaut.http.MediaType;
+import io.micronaut.http.annotation.Body;
+import io.micronaut.http.annotation.Controller;
+import io.micronaut.http.annotation.Get;
+import io.micronaut.http.annotation.Post;
+import io.swagger.v3.oas.annotations.Hidden;
+import java.net.URI;
+
+@Controller("/ws/traveltime")
+public class TravelTimeController {
```

Comment:
You should add in a basic comment this the required controller class for micronaut to represent the service or something...